src/coreclr/inc/corjitflags.h

```diff
@@ -217,10 +217,32 @@ class CORJIT_FLAGS
         return instructionSetFlags.GetInstructionFlagsFieldCount();
     }
 
+    uint32_t GetMaxVectorTBitWidth()
+    {
+        return maxVectorTBitWidth;
+    }
+
+    void SetMaxVectorTBitWidth(uint32_t value)
+    {
+        maxVectorTBitWidth = value;
+    }
+
+    uint32_t GetPreferredVectorBitWidth()
+    {
+        return preferredVectorBitWidth;
+    }
+
+    void SetPreferredVectorBitWidth(uint32_t value)
+    {
+        preferredVectorBitWidth = value;
+    }
```

Comment:
Seems like these shouldn't be part of CORJIT_FLAGS; they should be returned to the JIT as part of a new JIT-EE method on ICorStaticInfo, maybe `getVectorInfo()`?

Reply:
Fixed.